.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+#### What? Why?
+
+
+#### What should we test?
+
+
+#### How does this relate to the Spree upgrade?
+
+[remove me if not needed]
+
+#### Discourse thread
+
+[remove me if not needed]
+
+#### Dependencies
```

Comment:
When the PR depends on another one in the same or any other repo. The case I see (which might not happen often) is to depend on https://github.com/openfoodfoundation/ofn-install or a new feature requiring a refactor to be shipped first.

Yes, it needs to be described better.